20-logging.py

```diff
@@ -31,10 +34,11 @@
     t_level = "DEBUG"
     level = logging.DEBUG
 # Set up WARNING and above as stderr, below that to stdout.
-handlers = [
-    logging.StreamHandler(level=logging.WARNING),
-    logging.StreamHandler(level=level, stream=sys.stdout),
-]
+warnhandler = logging.StreamHandler(stream=sys.stderr)
+warnhandler.setLevel(logging.WARNING)
+lowhandler = logging.StreamHandler(stream=sys.stdout)
+lowhandler.setLevel(level)
+handlers = [warnhandler, lowhandler]
 if customlogger:
     forward_lsst_log(t_level)
     handlers = [IPythonHandler()]
```

Comment:
Does this code work? How does it know that we want to use INFO logging here? Previously we called logging.basicConfig with a level parameter that was removed in #84.

Reply:
We want to use INFO unless the env. var DEBUG is set in which case we want to use DEBUG.  It does work--this is the change I just made for Merlin because you can't set the level in the StreamHandler constructor; rather, you have to set it on an extant handler.

Reply:
t_level is the string you need to pass to the forward lsst log function so that the C++ LSST Loggers get forwarded to python with the right level.

Reply:
I don't understand how this is working because logging.basicConfig defaults to WARNING so how do it know to print all the INFO messages? Is IPythonHandler defaulting to INFO internally or something?

Reply:
Rewritten--I think it's behaving correctly now.